app/migration_v_0_7.go

```diff
@@ -0,0 +1,24 @@
+package app
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Here we need to make all the links the
+func (app *App) Migration07(ctx sdk.Context) {
+	resourceList := app.resourceKeeper.GetAllResources(&ctx)
```

Comment:
We need to:
* Firstly, iterate without loading everything in memory and completely erase previous/next links;
* Next, iterate without loading everything in memory and call  `GetLastResourceVersionHeader`;

Reply:
Ok, will do it later.